WcaOnRails/lib/solve_time.rb

```diff
@@ -27,8 +27,7 @@ def wca_value=(wca_value)
     @time_centiseconds = nil
 
     if @event.fewest_moves?
-      # The average field for 333fm is pretty weird. It's the sum
-      # of the solves, multiplied by 100.
+      # The average field for 333fm is stored the same way as for other events, multiplied by 100.
```

Comment:
Oh ha, that was totally wrong. This change sounds good, or maybe "...is the average of the solves multiplied by 100"